src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -2702,12 +2665,9 @@ static TestResult DoTest(in Args args, int currentPid)
         Debug.Assert(SimpleRefPayLoad.NumPinned == SimpleRefPayLoad.NumUnpinned);
 #endif
 
+        Console.WriteLine($"Time Taken: {tEnd - tStart}ms");
         sw.WriteLine("Took {0}ms", tEnd - tStart);
         sw.Flush();
-
-        // sw.WriteLine("after init: heap size {0}, press any key to continue", GC.GetTotalMemory(false));
-        // Console.ReadLine();
-
```

Comment:
again, we shouldn't get rid of this. this is useful when you want to have the process paused at the end instead of just exit. could you turn this into a config.